packages/@okta/i18n/src/properties/login_nl_NL.properties

```diff
@@ -867,7 +867,7 @@ consent.required.privacyPolicy = Privacybeleid
 consent.required.consentButton = Toegang Verlenen
 consent.required.cancelButton = Niet toestaan
 # {0} is an app name
-granular.consent.scopes.title = <$1>{0}</$1> <$2>vraagt toegang tot uw account</$2>
+granular.consent.scopes.title = <$1>{0}</$1> <$2>wants to access your account</$2>
```

Comment:
English here :(

Reply:
@pengyao-okta if you could file a follow up 🙇‍♀️

Reply:
it looks like the original english one was: 
`granular.consent.scopes.title = <$1>{0}</$1> <$2>requests access to your account</$2>`

and then we changed it to:
`granular.consent.scopes.title = <$1>{0}</$1> <$2>wants to access your account</$2>`

so it's possible for some reason XTM is doing a reanalysis on these?

Reply:
Asking linguist to make the fix.